docs/conditions.md

```diff
@@ -34,10 +34,16 @@ The value can be given as a Python `True` or `False` Boolean value or as the str
 
 ### `parameter`
 
-Allows a flag to be enabled based on a GET parameter with the name given as the condition's value.
+Allows a flag to be enabled by including a parameter in the request's query string. `value` is the name of the parameter, or a name and expected value. If an expected value isn't provided, the value must by `true`.
```

Comment:
Typo: "must by"

Reply:
This says the value must be `true`, but below the default seems to be `True`.

Reply:
Correct! Thanks for catching.

Reply:
Corrected in 70bc29a